be/src/exec/pipeline/exchange/multi_cast_local_exchange.cpp

```diff
@@ -150,8 +154,10 @@ void MultiCastLocalExchanger::_update_progress(Cell* fast) {
         Cell* t = _head->next;
         if (t == nullptr) break;
         delete _head;
+        _current_memory_usage -= _head->memory_usage;
         _head = t;
     }
+    _current_row_size = _current_accumulated_row_size - _head->accumulated_row_size;
```

Comment:
What is current_row_size used for

Reply:
How many rows are buffered. I don't know , maybe it's a good indicator for spilling.